File: EZBUILD/app/src/main/java/hr/itrojnar/ezbuild/model/messaging/constructionSite/UpdateConstructionSite.kt
package hr.itrojnar.ezbuild.model.messaging.constructionSite

import android.os.Parcelable
import hr.itrojnar.ezbuild.model.viewModels.ConstructionSite
import kotlinx.parcelize.Parcelize

@Parcelize
data class UpdateConstructionSiteRequest (
    var idConstructionSite             : Int,
    var base64Image                    : String,
    var fullAddress                    : String,
    var latitude                       : Double,
    var longitude                      : Double,
    var isActive                       : Boolean,
    var constructionSiteManagerID      : Int,
    var employees                      : ArrayList<Int>
) : Parcelable

@Parcelize
data class UpdateConstructionSiteResponse(
    var constructionSite  : ConstructionSite,
    var isSuccessful      : Boolean,
    val message           : String
) : Parcelable

File: EzBuildWebAPI/EzBuildServicesMessaging/ConstructionSites/UpdateConstructionSite.cs
using EzBuildServicesMessaging.Base;
using EzBuildServicesMessaging.ViewModels;
using System.Collections.Generic;

namespace EzBuildServicesMessaging.ConstructionSites
{
    public class UpdateConstructionSiteRequest : BaseRequest
    {
        public int IDConstructionSite { get; set; }
        public string Base64Image { get; set; }
        public string FullAddress { get; set; }
        public decimal Latitude { get; set; }
        public decimal Longitude { get; set; }
        public int? ConstructionSiteManagerID { get; set; }
        public IEnumerable<int> Employees { get; set; }
        public bool IsActive { get; set; }
    }

    public class UpdateConstructionSiteResponse : BaseResponse
    {
        public ConstructionSiteVM ConstructionSite { get; set; }
    }
}
